Formula/autoenv.rb

```diff
@@ -16,4 +16,11 @@ def caveats; <<-EOS.undent
       source #{opt_prefix}/activate.sh
     EOS
   end
+
+  test do
+    (testpath/"test").mkdir
+    (testpath/"test/.env").write "echo it works"
+    testcmd = "yes | bash -c '. #{opt_prefix}/activate.sh; autoenv_cd test'"
```

Comment:
I think `shell_output` makes much more sense here, because you should treat the entire thing as a test command, rather than, this part is the test command, that part is the input we feed into it (a y to answer a prompt...).